bundles/org.openhab.binding.opengarage/src/main/java/org/openhab/binding/opengarage/internal/OpenGarageHandler.java

```diff
@@ -57,20 +62,25 @@ public OpenGarageHandler(Thing thing) {
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
         try {
-            logger.warn("Received command {} for thing '{}' on channel {}", command, thing.getUID().getAsString(),
+                logger.warn("Received command {} for thing '{}' on channel {}", command, thing.getUID().getAsString(),
                     channelUID.getId());
-            switch (channelUID.getId()) {
-                case OpenGarageBindingConstants.CHANNEL_OG_STATUS:
-                    if (command instanceof OnOffType) {
-                        changeStatus(((OnOffType) command).equals(OnOffType.ON));
-                        return;
-                    }
-                    break;
-                default:
+                switch (channelUID.getId()) {
+                    case OpenGarageBindingConstants.CHANNEL_OG_STATUS:
+                    case OpenGarageBindingConstants.CHANNEL_OG_STATUS_SWITCH:
+                    case OpenGarageBindingConstants.CHANNEL_OG_STATUS_ROLLERSHUTTER:
```

Comment:
This is a small nit, but do you need a switch statement at all? It only has one outcome.  i would think you could simply remove the switch statement and simple and just jump straight to checking the command types.

Reply:
I guess we could do this, I think this is safer as we at least make sure the right channel is receiving the command?

Reply:
Sounds good to me.